src/Components/Inquiry/Views/InquiryInquiry.tsx

```diff
@@ -45,7 +46,7 @@ const InquiryInquiry: React.FC<InquiryInquiryProps> = ({ artwork }) => {
   const { submitArtworkInquiryRequest } = useArtworkInquiryRequest()
 
   const handleTextAreaChange = ({ value }: { value: string }) => {
-    if (mode === "Confirm" && value !== inquiry.message) {
+    if (mode === "Confirm" && !AUTOMATED_MESSAGES.includes(inquiry.message)) {
```

Comment:
this is not correct, we should check for `value` instead of `inquiry.message`. Check how it was on https://github.com/artsy/force/pull/12444/files#diff-a2d236f2e3c51db43465ff1c391d11cf95a825e2e91366b0123e35df3636b10bR48